packages/sdk/src/extensions/external-positions/instances/kiln.ts

```diff
@@ -19,12 +26,34 @@ export const kilnStakeArgsEncoding = [
   },
 ] as const;
 
+export const kilnClaimFeesArgsEncoding = [
+  {
+    type: "address",
+    name: "stakingContract",
+  },
+  {
+    name: "publicKeys",
+    type: "bytes[]",
+  },
+  {
+    name: "claimType",
+    type: "string",
```

Comment:
claimType is enum, not a string. And the name is `claimFeeType`
![Uploading Screenshot 2023-08-29 at 09.49.03.png…]()